Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -52,36 +59,56 @@ public class FileLogger: Logging {
     }
 
     var currentLogFileUrl: URL {
-        suiteName == nil ?
-            logDirURL.appendingPathComponent("\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension) :
-            logDirURL.appendingPathComponent("extension-\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension)
+        if let appName = appName {
+            return logDirURL
+                .appendingPathComponent("\(appName)-\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        } else {
+            return logDirURL
+                .appendingPathComponent("\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        }
     }
 
     public var levels: [Level] = [.info]
 
     // MARK: - Initializers
 
+    /// This is public constructor for FileLogger.
+    /// - Parameters:
+    ///   - appName: Name of the App used for appendingPathComponent of logDirURL.
+    ///   - appGroupID: Distinguishes log files according to the app environment (main app, extension 1, extension 2, etc...).
+    ///   - externalLogger: logger for error handling
+    ///   - logDirectoryName: value for appendingPathComponent in fileManager
+    ///   - fileHeaderContent: initialContent value for creating log files
+    ///   - numberOfLogFiles: Total count of logfiles (value 4 means four days of each file persistence).
```

Comment:
Proc mas logfiles takhle dohromady? To co mas v zavorce nedava anglicky smysl. Obecne jsou ty komentare dost odflaknuty. Podival bych se poradne k cemu jsou, pochopil je a pak to napsal tak, aby kdyz si to bude cist nekdo cizi, tak mu to pomohlo.